apps/web/components/booking/BookingListItem.tsx

```diff
@@ -424,7 +424,7 @@ function BookingListItem(booking: BookingItemProps) {
       <RescheduleDialog
         isOpenDialog={isOpenRescheduleDialog}
         setIsOpenDialog={setIsOpenRescheduleDialog}
-        bookingUId={booking.uid}
+        bookingUid={booking.uid}
```

Comment:
Throughout the codebase we use `bookingUid` so cleaning this up to be more consistent.